### ISSUE TITLE
WISH: Connect PVs to WISH Jaws OPI

### ISSUE THREAD

**Issue by daryakoskeroglu** [ISISComputingGroup/IBEX#6917]
As a developer I would like to connect missing PVs on WISH Jaws OPI.

Please check the notes on PR for the OPI: https://github.com/ISISComputingGroup/ibex_gui/pull/1379

The PVs that are need to be connected are:

 1. Experiment Type
The list needs to be populated (Powder Diffraction, Single Crystal and Other). Possible solution is using the motion setpoints and 
lookup file system,
Motion Setpoints Wiki: https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/Motion-Set-points
Lookup File System Wiki: https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/ISIS-modules-for-file-handling#readascii
 2. Resolution
 3. Dead Band
 4. Start/Stop and Set buttons

### Acceptance Criteria
- [ ] The PVs are connected to relevant widgets

### Extra Information
VI for the WISH Jaws is located here:  C:\LabVIEW Modules\Instruments\WISH\WISH Jaws

Ticket for creating the OPI: https://github.com/ISISComputingGroup/IBEX/issues/6033

To check the WISH Instrument wiki: https://github.com/ISISComputingGroup/IBEX/wiki/WISH-Instrument-Details

The jaws are Galil controlled.

GEM & POLARIS have similar OPIs for controlling their jaws.

**Comment by JamesKingWork** [comment]
Is this connecting the PVs or writing a new IOC to manage the WISH jaws?

**Comment by JamesKingWork** [comment]
Proposed as the work was expected to be finished in https://github.com/ISISComputingGroup/IBEX/issues/6033 so probably assumes the same priority as that ticket

**Comment by daryakoskeroglu** [comment]
> Is this connecting the PVs or writing a new IOC to manage the WISH jaws?

A bit of both, there is an IOC for Jaws Manager which can be found in `C:\Instrument\Apps\EPICS\support\jaws` but extra PVs should be implemented in order to finish WISH Jaws manager, for example; a PV which can populate the Experiment Type menu

**Comment by rerpha** [comment]
ok, so I think we can probably not worry too much about deadband/motor res as those are things you can set in the table of motors anyway and they probably won't change (?) very often as they're configured using phsyical switches in the galil rack. 

activity LED can also be removed as it's a SECI VI thing which we don't bother adding in anymore 

start/stop/set can also probably be removed as the motion setpoints stuff will do this I think

which means I think this ticket is just to set up motion setpoints for the jaws. I am happy to assist with whoever takes this ticket on as i've just done it for the NIMROD camera, albeit only on one axis.

**Comment by rerpha** [comment]
they dont need motion setpoints so closing this ticket - WISH are happy with their jaws currrently
